app/src/main/java/net/gsantner/markor/activity/DocumentEditFragment.java

```diff
@@ -470,14 +464,14 @@ public boolean onOptionsItemSelected(final MenuItem item) {
             case R.id.action_share_html_source: {
                 if (saveDocument(false)) {
                     TextConverter converter = TextFormat.getFormat(_document.getFormat(), activity, _document).getConverter();
-                    _shareUtil.shareText(converter.convertMarkup(_hlEditor.getText().toString(), _hlEditor.getContext(), false, _document.getFile()),
```

Comment:
`_hlEditor.getText().toString()` was generating a bunch of warnings everywhere. Moved it to a single function in order to minimize noise